admins/pageflow/entry_templates.rb

```diff
@@ -0,0 +1,66 @@
+module Pageflow
+  ActiveAdmin.register EntryTemplate, as: 'EntryTemplate' do
+    menu false
+    config.batch_actions = false
+    actions :new, :create, :edit, :update
+    form partial: 'form'
+
+    belongs_to :account, parent_class: Pageflow::Account
+
+    breadcrumb do
+      breadcrumb_links.first 3
+    end
+
+    controller do
+      helper Pageflow::Admin::FormHelper
+      helper Pageflow::Admin::WidgetsHelper
+
+      def new
+        @page_title = page_title(params[:entry_type])
+        super
+      end
+
+      def create
+        create! { redirect_path }
+      end
+
+      def edit
+        @page_title = page_title(resource.entry_type)
+        super
+      end
+
+      def update
+        update! { redirect_path }
+      end
+
+      private
+
+      def page_title(entry_type_name)
+        "#{I18n.t('pageflow.admin.entry_templates.page_title')} "\
+        "#{I18n.t('activerecord.values.pageflow/entry.type_names.' + entry_type_name)}"
+      end
+
+      def permitted_params
+        params.permit(entry_template: permitted_entry_template_attributes)
+      end
+
+      def permitted_entry_template_attributes
+        [
+          :account_id,
```

Comment:
Is this needed? If yes, we should have a spec that ensures a user cannot create/update entry templates of other accounts.